src/imgui/ImGuiSettings.cc

```diff
@@ -374,7 +374,7 @@ void ImGuiSettings::showMenu(MSXMotherBoard* motherBoard)
 				ImGui::TextUnformatted("Select layout"sv);
 				auto names = getExistingLayouts();
 				if (auto selectedLayout = listExistingLayouts(names)) {
-					const auto& [name, displayName] = *selectedLayout;
+					const auto& name = *selectedLayout;
```

Comment:
(I only read this PR very quickly, I did not test anything).

Is this correct? If I read the code correctly, then this now uses the `displayName`. But I think `manager.loadIniFile` should be the `fullName`, no?